_component_design/vertical-stepper.md

```diff
@@ -11,7 +11,7 @@ status: stable
 last-modified: 2018-08-21
 helix-ui-css: true
 helix-ui-javascript: true
-pagelink: https://rackerlabs.github.io/helix-ui/components/accordions/
+pageLink: components/accordion/
```

Comment:
Given that the Stepper is still in beta at `components/stepper/` (not in component explorer navigation), I'd recommend changing this to `none` until it's brought out of beta.

Reply:
👍